CHANGELOG.adoc

```diff
@@ -19,6 +19,7 @@
 - https://github.com/eclipse-syson/syson/issues/192[#192] [import] Fix an issue where the /* and */ of a Comment's body were imported while importing a textual SysML file. 
 - https://github.com/eclipse-syson/syson/issues/188[#188] [import] Fix an issue where some Memberships were contained in their parent through `ownedRelatedElement` instead of `ownedRelationship` reference.
 - https://github.com/eclipse-syson/syson/issues/199[#199] [explorer] Fix an issue where the rename action was not renaming tree items anymore
+- https://github.com/eclipse-syson/syson/issues/209[#209] [diagrams] EnumerationDefinition is created without any name
```

Comment:
Please change to `EnumerationDefinition was created without any name`

Reply:
Done